packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### New Features
```

Comment:
Not "Enhancements"? (Enhancement == new feature)

Reply:
My reasoning was "New API" (stable `Navigator`) => "New feature" section

Although I'm very much fine putting it under "Enhancements"

Reply:
Done https://github.com/WordPress/gutenberg/pull/64613/commits/af97de9356dd7a8ba9a3ceeef705edebc0d36e71